### PR TITLE
Changelog 57

### PR DESCRIPTION
**Pull request by mfeifer**
5 PRs

[skip ci]

### CODE REVIEW

**Comment by mfeifer** [comment]
@miq-bot add_label wip

**Comment by mfeifer** [comment]
@miq-bot add_label documentation

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,18 @@
 
 All notable changes to this project will be documented in this file.
 
-## Unreleased - as of Sprint 56 end 2017-02-27
+## Unreleased - as of Sprint 57 end 2017-03-27
+
+### Added
```
Comment (gmcculloug) [inline CHANGELOG.md]: @mfeifer This looks good.  Just need to add PR https://github.com/ManageIQ/manageiq-content/pull/76.

Comment (mfeifer) [inline CHANGELOG.md]: 👍

**Comment by mfeifer** [comment]
@miq-bot remove_label wip

**Pull request merged**